lib/bank_account/account.rb

```diff
@@ -0,0 +1,37 @@
+module BankAccount
+
+  class Account
+    attr_reader :id, :balance # attributes' default instance method
+
+    #class method
+    def initialize(id, initial_balance)
+      @id = id
+      @balance = initial_balance #local variable
+      # made initial balance equal to instance variable balance, which actually turns out to be the current balance
```

Comment:
My thoughts are that right now, your code is readable enough that you don't need this comment!  Your variable names are great and make a lot of sense in the context here.